### PR TITLE
replace deprecated nose with pytest

### PR DESCRIPTION
**Pull request by pgajdos**
`nose` will stop working with future python versions.

### CODE REVIEW

File: tests/test_files.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from io import StringIO
 from unittest.mock import patch
 
-from nose.tools import eq_
+import pytest
```
Comment (chfw) [inline tests/test_files.py]: is it required to `import pytest` here?

without pytest, would the unit tests fail?

the reason I flag this up is: my editor will underline it, and say: unused imports

Comment (pgajdos) [reply tests/test_files.py]: No, it is not required, fixed.

Comment (pgajdos) [reply tests/test_files.py]: There are unnecessary `import pytest` in moban, too, should I care of it?

Comment (chfw) [reply tests/test_files.py]: yes, please!

**Review by chfw — APPROVED** [review]
LGTM

**Pull request merged**